joint_limits:
  joint_1:
    has_velocity_limits: true
    max_velocity: 0.261799383
  joint_5:
    has_velocity_limits: true
    max_velocity: 0.261799383
  joint_4:
    has_velocity_limits: true
    max_velocity: 0.261799383
  joint_2:
    has_velocity_limits: true
    max_velocity: 0.261799383
  joint_3:
    has_velocity_limits: true
    max_velocity: 0.261799383
  joint_6:
    has_velocity_limits: true
    max_velocity: 0.261799383